README.md

```diff
@@ -1,36 +1,128 @@
 # Arlo's Commit Notation
 
-## Upper case: May change behavior
+This commit notation allows developers to convey 2 critical pieces of metadata about each commit:
 
-| Prefix  | Meaning                                                   |
-| ------- | --------------------------------------------------------- |
-| F       | Feature (<= 8 <abbr title="lines of code">LoC</abbr><sup>[4]</sup>)                        |
-| B       | Bug (<= 8 <abbr title="lines of code">LoC</abbr><sup>[4]</sup>)                              |
-| R       | Test-supported Procedural Refactoring<sup>[3]</sup>                      |
+1. How risky is it? What has the original author done to mitigate risk?
+2. What was the intention? When the original author changed the code, what was s/he attempting to accomplish?
 
-## Lower case: Low risk
+This information is conveyed in the first 3 characters of the commit summary line. That way a receiving developer can quickly scan the commit log in order to determine risk and intent for any incoming change set.
 
-| Prefix  | Meaning                                                      |
-| ------- | ------------------------------------------------------------ |
-| t       | Test only                                                    |
-| d       | Developer documentation changes (not end-user facing<sup>[1]</sup>)        |
-| a       | Automated formatting / generation                               |
-| r       | Provable Refactoring<sup>[2]</sup>                 |
-| c       | Comments (add/delete)                                        |
-| e       | Environment (non-code) changes that affect development setup, and other tooling changes that don't affect program behavior (e.g. linting) |
+This is particularly useful when:
 
-## Three characters: The danger zone!
+1. Deciding whether to approve a pull request.
+2. Reading `main` &mdash; just the pull request commit summaries to understand the history of changes for a release.
 
-| Prefix  | Meaning                                                   |
-| ------- | --------------------------------------------------------- |
-| F!!     | Feature (> 8 <abbr title="lines of code">LoC</abbr><sup>[4]</sup>)                             |
-| B!!     | Bug (> 8 <abbr title="lines of code">LoC</abbr><sup>[4]</sup>)                                   |
-| R!!     | Non-provable refactoring                                  |
-| ***     | Does not compile intermediate step                        |
+## The Four Risk Levels
 
-# Description
+| Risk Level | Code | Example | Meaning |
+| --- | --- | --- | --- |
+| **Known safe** | lowercase letter | `r   Extract method Applesauce` | Addresses all known and unknown risks. |
```

Comment:
It a lot of UI `   ` doesn't format as three spaces, it collapses to one. I wish for a notation that doesn't have that problem.

In one project we use `r - ` instead. Maybe riff on that?

This problem may be out of scope for the current pull request.

Reply:
![image](https://user-images.githubusercontent.com/1259628/124042890-b2b2ad80-d9be-11eb-8ddb-20929e065403.png)

Reply:
That's a good idea. I think it warrants its own PR / discussion.

Reply:
https://github.com/RefactoringCombos/ArlosCommitNotation/issues/26